### DISCUSSION TITLE
WebWorker error on importScripts

### DISCUSSION THREAD

**Discussion by tiagooliveira08** [vercel/next.js#32533] [Help]
### What example does this report relate to?

with-web-worker

### What version of Next.js are you using?

12.0.7

### What version of Node.js are you using?

14

### What browser are you using?

edge

### What operating system are you using?

ubuntu

### How are you deploying your application?

next start

### Describe the Bug

When trying to import a specific script, I get an error, the interesting thing is that I run another script that is excelJS and it works.

### Expected Behavior

the brazilian utils script should be imported without showing the error on the console.

Uncaught DOMException: Failed to execute 'importScripts' on 'WorkerGlobalScope': The script at 'https://unpkg.com/@brazilian-utils/brazilian-utils@1.0.0-rc.12/dist/brazilian-utils.cjs.production.min.js' failed to load.

![image](https://user-images.githubusercontent.com/27312393/146070592-b99f21d8-b9af-41f4-a169-3aa624b3cfd5.png)

### To Reproduce
access https://github.com/tiagooliveira08/web-worker-error-example-import
Clone repository
run npm install && npm run dev

**Comment by nikitapilgrim** [comment]
+1
